pydatastructs/trees/heaps.py

```diff
@@ -18,6 +18,44 @@ class Heap(object):
 
 
 class DHeap(Heap):
+     def heapsort( aList ):

```

Comment:
The method should have been placed after all the methods. Do not place methods before the doc strings.

Reply:
```suggestion
     def heapsort(self, aList):
```